generator/templates/server/server.gotmpl

```diff
@@ -361,51 +363,58 @@ func (s *Server) Serve() (err error) {
 		}
 		httpsServer.Handler = s.handler
 
-    // Inspired by https://blog.bracebin.com/achieving-perfect-ssl-labs-score-with-go
+		// Inspired by https://blog.bracebin.com/achieving-perfect-ssl-labs-score-with-go
 		httpsServer.TLSConfig = &tls.Config{
 			// Causes servers to use Go's default ciphersuite preferences,
 			// which are tuned to avoid attacks. Does nothing on clients.
 			PreferServerCipherSuites: true,
 			// Only use curves which have assembly implementations
 			// https://github.com/golang/go/tree/master/src/crypto/elliptic
-			CurvePreferences: []tls.CurveID{tls.CurveP256},{{ if .UseModernMode }}
-   		// Use modern tls mode https://wiki.mozilla.org/Security/Server_Side_TLS#Modern_compatibility
+			CurvePreferences: []tls.CurveID{tls.CurveP256},
+		{{- if .UseModernMode }}
+			// Use modern tls mode https://wiki.mozilla.org/Security/Server_Side_TLS#Modern_compatibility
 			NextProtos: []string{"http/1.1", "h2"},
 			// https://www.owasp.org/index.php/Transport_Layer_Protection_Cheat_Sheet#Rule_-_Only_Support_Strong_Protocols
 			MinVersion: tls.VersionTLS12,
-      // These ciphersuites support Forward Secrecy: https://en.wikipedia.org/wiki/Forward_secrecy
+			// These ciphersuites support Forward Secrecy: https://en.wikipedia.org/wiki/Forward_secrecy
 			CipherSuites: []uint16{
 				tls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
```

Comment:
can you add 

```
tls.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,
tls.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,
```

to these ciphersuites since we're no longer supporting 1.7

Reply:
ok

Reply:
done